specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-10-01-preview/Entities.json

```diff
@@ -1377,6 +1377,18 @@
         "alertType": {
           "type": "string",
           "description": "The name of the alert type."
+        },
+        "intent": {
```

Comment:
>intent [](http://example.com/codeflow?start=9&length=6)

You already have a free-form description field. I'd recommend considering if/how you could constrain this string to a set of known enum values (make it a string enum). Otherwise, it's likely to turn into a second description string.

Reply:
Good point. done, thanks.